store.go

```diff
@@ -24,7 +23,7 @@ const (
 )
 
 var OutStoreFormat = UNSET
-var TemplateFilter func(data []byte) []byte
+var TemplateFilter func(data []byte) ([]byte, error)
```

Comment:
The changes around the `TemplateFilter` and the rest of the `store.go` and `template.go` will conflict with another PR #534. @aelsabbahy #534 is ready to be merged and maybe it should so @ripienaar makes the right modifications to his code. Funny enough I ended up tweaking my code to make it easier for Package usage as well.

Reply:
ok, once you merge that one I'll rebase here

Reply:
Merged #534